python/cuml/cuml/metrics/_classification.py

```diff
@@ -13,16 +13,121 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import warnings
 
-from cuml.internals.input_utils import input_to_cupy_array
 import cuml.internals
+from cuml.internals.input_utils import input_to_cupy_array
```

Comment:
What is doing this (re)sorting of import statements? My internal linter agrees with this sort, so mostly this question is "don't we have `ruff` or similar to take care of getting import statements correctly sorted?"

Reply:
I've been doing these with my editor as I touch files (if the reorg is small enough). We don't currently run `isort` on anything, nor do we `ruff`. `ruff` wouldn't catch all issues anyway since it doesn't currently work on cython and too much of `cuml` is currently cython. Future todo.